samigo/samigo-app/src/webapp/jsf/delivery/item/deliverFillInNumeric.jsp

```diff
@@ -24,6 +24,9 @@ should be included in file importing DeliveryMessages
 **********************************************************************************/
 --%>
 -->
+<script>includeWebjarLibrary("qtip2");</script>
+<script src="/library/webjars/bootstrap/3.3.7/js/bootstrap.min.js<h:outputText value="#{delivery.CDNQuery}" />"></script>

```

Comment:
can't you repeat the pattern of line 27?

<script>includeWebjarLibrary('bootstrap');</script>

Reply:
Yeah, that works and certainly looks more elegant.

Reply:
@ottenhoff - I'm observing that the bootstrap CSS is overriding the Sakai styling, changing the base font styles, etc. I recommend sticking with my first commit, which I can force push to this PR. Let me know how you'd like me to proceed.